### ISSUE TITLE
`e3sm_to_cmip` breaks with vertical dimension name with changes in `nco=5.14` 

### ISSUE THREAD

**Issue by tomvothecoder** [E3SM-Project/e3sm_to_cmip#184]
> It turned out that the NCO version different changed the vertical dimension name, which resulted that error you found. I post this on the “nco” channel to see if Charlie has some suggestion. Below is what I posed on slack:

> - here is a minor glitch [@Mingxuan Wu](https://urldefense.us/v3/__https:/gcc02.safelinks.protection.outlook.com/?url=https*3A*2F*2Facmeclimate.slack.com*2Fteam*2FU04MLNV8C95&data=05*7C01*7Cmingxuan.wu*40pnnl.gov*7Ca2639002646d4459331c08db096fe7a2*7Cd6faa5f90ae240338c0130048a38deeb*7C0*7C0*7C638114148469991660*7CUnknown*7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0*3D*7C3000*7C*7C*7C&sdata=QJJ*2F9yOctw1BAYFzSsgC*2BW9*2BsMGqTC7TaU*2BcY8j1OlA*3D&reserved=0__;JSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUlJQ!!G2kpM7uM-TzIFchu!lrCdlNcNG-e1z2TXUis4bJVcnlwxPXts6QU9FwnS95lJ6cfx_j1kccmU5MFStT0WyA$) and I are using e3sm_to_cmip to convert some atmospheric variables. When converting some 3D variables, we noticed the NCO vertically remaped files ends with dimension name lev using NCO 5.1.4 from e3sm_unified, and plev using 5.1.3, which is included in e3sm_to_cmip dev enviroment. The mapping file [vrt_remap_plev19.nc](https://urldefense.us/v3/__https:/gcc02.safelinks.protection.outlook.com/?url=http*3A*2F*2Fvrt_remap_plev19.nc*2F&data=05*7C01*7Cmingxuan.wu*40pnnl.gov*7Ca2639002646d4459331c08db096fe7a2*7Cd6faa5f90ae240338c0130048a38deeb*7C0*7C0*7C638114148469991660*7CUnknown*7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0*3D*7C3000*7C*7C*7C&sdata=*2BvYgP7zJSWFTpFROpZ1IuXhRm0qV7*2F*2FL*2FpBJoymxJGY*3D&reserved=0__;JSUlJSUlJSUlJSUlJSUlJSUlJSUlJSUl!!G2kpM7uM-TzIFchu!lrCdlNcNG-e1z2TXUis4bJVcnlwxPXts6QU9FwnS95lJ6cfx_j1kccmU5MF-wlvimA$) does have plev as the dimension. I'm wondering if this is expected for the newer versions of NCO, if in this case, we perhaps need to update e3sm_to_cmip or its workflow for this change.
>
> -- Originally from an email chain with @chengzhuzhang, @mingxuanwupnnl, and @TonyB9000

This was an inadvertent change to the default setting (regression).

### Current Workaround
The current workaround is to install `nco=5.1.3` explicitly with `e3sm_to_cmip=1.9.1`.

Example:
```bash
conda create -n e2c -c conda-forge e3sm_to_cmip=1.9.1 nco=5.1.3
conda activate e2c
```

### Future Solution

Wait for `nco=5.1.5` which should include the default as `plev` again.

**Comment by tomvothecoder** [comment]
@TonyB9000 and @chengzhuzhang, was this ever fixed in `nco>=5.1.5`? If so, I will close this issue.

**Comment by chengzhuzhang** [comment]
I believe so.
